### PR TITLE
Adaptive card with people picker nodejs

### PR DESCRIPTION
**Pull request by NikitaJain-MSFT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |11| Channel messages with RSC permissions | Demonstrating on how a bot can receive all channel messages with RSC without @mention. | [View][messageswithrsc#csharp] |[View][messageswithrsc#js]
 |12| Bot with SharePoint file to view in Teams file viewer | This sample demos a bot with capability to upload files to SharePoint site and same files can be viewed in Teams file viewer. |[View][botwithsharepointfileviewer#csharp]|[View][botfileviewer#js]
 |13|  Type ahead search control on Adaptive Cards | This sample shows the feature of type ahead search (static and dynamic) control in Adaptive Cards. |[View][typeaheadsearch#csharp]|[View][typeaheadsearchonadaptivecard#js]
+|14|  People picker control on Adaptive Cards | This sample shows the feature of people picker control in Adaptive Cards. ||[View][peoplepickeronadaptivecard#js]
```
Comment (AbhijeetThakur-msft) [inline README.md:82]: People picker control in Adaptive Cards

File: samples/bot-people-picker-adaptive-card/nodejs/Readme.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    ```
+5) Open the `.env` configuration file in your project folder (or in Visual Studio Code) and update the `MicrosoftAppId` and `MicrosoftAppPassword` with your app's base url. (Note the ClientId is the AppId created in step 1 (Setup for Bot), the ClientSecret is referred to as the "client secret" in step 1 (Setup for Bot) and you can always create a new client secret anytime.)
+
+6) Run your app
+
+    ```bash
+    npm start
+    ```
+7) Manually update the manifest.json
+    - Edit the `manifest.json` contained in the  `appPackage/` folder to replace with your MicrosoftAppId (that was created in step1.1 and is the same value of MicrosoftAppId in `.env` file) *everywhere* you see the place holder string `{{Microsoft-App-Id}}` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
```
Comment (AbhijeetThakur-msft) [inline samples/bot-people-picker-adaptive-card/nodejs/Readme.md:71]: There is no step as 1.1, please correct this one.

File: samples/bot-people-picker-adaptive-card/nodejs/Readme.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    - Zip up the contents of the `appPackage/` folder to create a `manifest.zip`
+    - Upload the `manifest.zip` to Teams (in the left-bottom *Apps* view, click "Upload a custom app")
+
+## Further reading
+
+- [Bot Framework Documentation](https://docs.botframework.com)
+- [Bot Basics](https://docs.microsoft.com/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0)
+- [Send Notification to User in Chat](https://docs.microsoft.com/en-us/graph/api/chat-sendactivitynotification?view=graph-rest-beta)
```
Comment (AbhijeetThakur-msft) [inline samples/bot-people-picker-adaptive-card/nodejs/Readme.md:79]: These reading links are not related to this sample.

File: samples/bot-people-picker-adaptive-card/nodejs/Readme.md (hunk 2 above)
Comment (AbhijeetThakur-msft) [inline samples/bot-people-picker-adaptive-card/nodejs/Readme.md:71]: Add details about what to put for {{domain-name}} as well.

**Review by AbhijeetThakur-msft — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NikitaJain-MSFT) [reply README.md]: updates

File: samples/bot-people-picker-adaptive-card/nodejs/Readme.md (hunk 3 above)
Comment (NikitaJain-MSFT) [reply samples/bot-people-picker-adaptive-card/nodejs/Readme.md]: added relevant links

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, approving it.

File: samples/bot-people-picker-adaptive-card/nodejs/appPackage/manifest.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            }
+          ]
+        }
+      ],
+      "isNotificationOnly": false
+    }
+  ],
+  "permissions": [ "identity", "messageTeamMembers" ],
+  "validDomains": [
+    "{{domain-name}}"
```
Comment (Wajeed-msft) [inline samples/bot-people-picker-adaptive-card/nodejs/appPackage/manifest.json:51]: I believe we don't need this domain name as we just have a Bot with Adaptive Card. Could you please check and remove placeholder?

**Review by Wajeed-msft — APPROVED** [review]

File: samples/bot-people-picker-adaptive-card/nodejs/appPackage/manifest.json (hunk 4 above)
Comment (AbhijeetThakur-msft) [reply samples/bot-people-picker-adaptive-card/nodejs/appPackage/manifest.json]: checked it and removed domain part.

**Review by AbhijeetThakur-msft — APPROVED** [review]
